scheduler/local.py

```
@@ -0,0 +1,381 @@
+        except sqlite3.OperationalError as exc:
+            log.debug("[scheduler] WAL skipped: %s", exc)
+        return db
+
+    def _init_db(self) -> None:
+        try:
+            db = self._connect()
+            db.executescript(_SCHEMA)
+            db.commit()
+            db.close()
+        except sqlite3.DatabaseError as exc:
+            log.error("[scheduler] schema init failed at %s: %s", self.path, exc)
```

Ruff (0.15.12)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)